linuxone/entrypoint.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/sh
+set -x
+
+DB_HOST=${DB_HOST:-postgresql}
+DB_PORT=${DB_PORT:-5432}
+REDIS_HOST=${REDIS_HOST:-redis}
+REDIS_PORT_NUM=${REDIS_PORT_NUM:-6379}
+GITLAB_HOST=${GITLAB_HOST:-gitlab-server}
+
+create_database_config() {
+cat <<EOF > /home/git/gitlab/config/database.yml
+#
+# PRODUCTION
+#
+production:
+  adapter: postgresql
+  encoding: unicode
+  database: gitlabhq_production
+  pool: 10
+  username: git
+  password: "${DB_PASS}"
+  host: ${DB_HOST}
+  port: ${DB_PORT}
+EOF
+}
+
+
+create_redis_config() {
+cat <<EOF > /home/git/gitlab/config/resque.yml
+production:
+  url: redis://${REDIS_HOST}:${REDIS_PORT_NUM}
+EOF
+}
+
+
+
+start_sshd() {
+if [ ! -f "/etc/ssh/ssh_host_rsa_key" ]; then
+        ssh-keygen -f /etc/ssh/ssh_host_rsa_key -N '' -t rsa
+fi
+if [ ! -f "/etc/ssh/ssh_host_dsa_key" ]; then
+        ssh-keygen -f /etc/ssh/ssh_host_dsa_key -N '' -t dsa
+fi
+
+if [ ! -d "/var/run/sshd" ]; then
+  mkdir -p /var/run/sshd
+fi
+
+/usr/sbin/sshd -E /var/log/sshd.log &
+}
+
+update_nginx_conf() {
+cp /home/git/gitlab/lib/support/nginx/gitlab /etc/nginx/conf.d/default.conf
+sed -ie 's/server unix:.*\.socket/server localhost:8181/' /etc/nginx/conf.d/default.conf
+sed -ie 's/events {/pid \/var\/run\/nginx.pid;\n\nevents {/' /etc/nginx/nginx.conf
+sed -ie "s/server_name YOUR_SERVER_FQDN/server_name ${GITLAB_HOST}/" /etc/nginx/conf.d/default.conf
+}
+
+
+update_gitlab_conf() {
+cd /home/git/gitlab/config
+sed -ie 's!default: /home/git/repositories!default: /home/git/data/repositories!g' gitlab.yml
+sed -ie "s/host: localhost/host: ${GITLAB_HOST}/" gitlab.yml
```

Comment:
Is there a reason we need to use multiple delimiters? If not, can we choose one delimiter, and stick with it? I'd propose `/`, since it is pretty standard.